src/config/dynamic-theme-fixes.config

```diff
@@ -10871,6 +10871,14 @@ html {
 
 ================================
 
+foragerproject.com
+
+INVERT
+.icon
```

Comment:
I don't want to go through whole page but maybe use a little bit more detailed selector to not invert all icons.
Change it to `.product-icons .icon` just to be sure.